### PR TITLE
translatable nestedform

### PR DESCRIPTION
**Pull request by der-On**
This is a follow up on https://github.com/wintercms/wn-translate-plugin/pull/30 as it was accidently closed bei github during a branch update.

### CODE REVIEW

**Comment by mjauvin** [comment]
Thé main branch needs to be merged into this branch, there's too much noise

**Comment by LukeTowers** [comment]
@mjauvin can you do it? You need to merge main into wip/translatable-nestedforms on this repo directly first

**Comment by mjauvin** [comment]
> @mjauvin can you do it? You need to merge main into wip/translatable-nestedforms on this repo directly first

@LukeTowers I just merged main into wip/translatable-nestedform but nothing has changed.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
+```bash
+composer require winter/wn-translate-plugin
+```
+
+Then, run the migrations to ensure the plugin is enabled:
+
+```bash
+php artisan winter:up
+```
+
```
Comment (LukeTowers) [inline README.md:18]:
```suggestion

If you are using a [public folder](https://wintercms.com/docs/setup/configuration#public-folder) then you will also need to regenerate it with `php artisan winter:mirror public --relative`
```

**Comment by LukeTowers** [comment]
@mjauvin apply my suggestion to regenerate the diff on github. Also in the future please ensure that allow edits from maintainers is checked when creating a PR so that I can apply them directly.

**Comment by mjauvin** [comment]
> @mjauvin apply my suggestion to regenerate the diff on github. Also in the future please ensure that allow edits from maintainers is checked when creating a PR so that I can apply them directly.

@LukeTowers I didn't create this PR, and can't commit to it either.

**Comment by LukeTowers** [comment]
@mjauvin whoops my bad, mistook StudioBosco for StudioAzura 🤦

**Comment by LukeTowers** [comment]
@der-On can you make this PR editable by maintainers and apply my suggestion?

**Comment by der-On** [comment]
> @der-On can you make this PR editable by maintainers and apply my suggestion?

Seem's like I cannot make it editable after it was created.

**Comment by bennothommo** [comment]
@der-On you can recreate the PR if you are able to extract the changes required, or rebase?

**Comment by mjauvin** [comment]
Can this one be closed @der-On ?

**Comment by der-On** [comment]
> Can this one be closed @der-On ?

Oh yes, please.

**Pull request closed without merging**